compiler/quilt/tools/build.py

```diff
@@ -216,7 +216,9 @@ def _build_node(build_dir, package, name, node, fmt, target='pandas', checks_con
                         assert isinstance(cachedobjs, list)
 
             # Check to see that cached objects actually exist in the store
-            if cachedobjs and all(os.path.exists(store.object_path(obj)) for obj in cachedobjs):
+            # TODO: check for changes in checks else use cache
+            # below is a heavy-handed fix but it's OK for check builds to be slow  
+            if not checks and cachedobjs and all(os.path.exists(store.object_path(obj)) for obj in cachedobjs):
```

Comment:
@kevinemoore pls review thx

Reply:
Looks good to me. We should eventually allow checked inputs to be cached, but it's fine to leave that as a TODO for now.